In-house patch to remove unnecessary dependencies from Nova's
requirements files. The specific reasons are as follows:

keystoneclient	Not applicable

kombu		Not applicable

lockfile	Not applicable

oslo.rootwrap	Not applicable to Solaris

oslo.vmware	Not applicable to Solaris (VMware specific)

paramiko	Not applicable to Solaris

posix_ipc	Not applicable

pycadf		Not applicable

suds		Not applicable to Solaris (VMware specific)

--- nova-2014.2.2/nova.egg-info/requires.txt.~1~	2015-02-05 06:33:48.000000000 -0800
+++ nova-2014.2.2/nova.egg-info/requires.txt	2015-02-25 00:59:17.326168445 -0800
@@ -7,7 +7,6 @@ decorator>=3.4.0
 eventlet>=0.15.1,<0.16.0
 Jinja2
 keystonemiddleware>=1.0.0
-kombu>=2.5.0
 lxml>=2.3
 Routes>=1.12.3,!=2.0
 WebOb>=1.2.3
@@ -16,9 +15,6 @@ PasteDeploy>=1.5.0
 Paste
 sqlalchemy-migrate==0.9.1
 netaddr>=0.7.12
-suds>=0.4
-paramiko>=1.13.0
-posix_ipc
 pyasn1
 Babel>=1.3
 iso8601>=0.1.9
@@ -26,18 +22,13 @@ jsonschema>=2.0.0,<3.0.0
 python-cinderclient>=1.1.0
 python-neutronclient>=2.3.6,<3
 python-glanceclient>=0.14.0
-python-keystoneclient>=0.10.0
 six>=1.7.0
 stevedore>=1.0.0  # Apache-2.0
 websockify>=0.6.0,<0.7
 wsgiref>=0.1.2
 oslo.config>=1.4.0  # Apache-2.0
 oslo.db>=1.0.0,<1.1  # Apache-2.0
-oslo.rootwrap>=1.3.0
-pycadf>=0.6.0
 oslo.messaging>=1.4.0,!=1.5.0,<1.6.0
 oslo.i18n>=1.0.0  # Apache-2.0
-lockfile>=0.8
 simplejson>=2.2.0
 rfc3986>=0.2.0  # Apache-2.0
-oslo.vmware>=0.6.0,<0.9.0               # Apache-2.0
--- nova-2014.2.2/requirements.txt.~1~	2015-02-05 06:26:51.000000000 -0800
+++ nova-2014.2.2/requirements.txt	2015-02-25 00:59:59.286958886 -0800
@@ -11,7 +11,6 @@ decorator>=3.4.0
 eventlet>=0.15.1,<0.16.0
 Jinja2
 keystonemiddleware>=1.0.0
-kombu>=2.5.0
 lxml>=2.3
 Routes>=1.12.3,!=2.0
 WebOb>=1.2.3
@@ -20,9 +19,6 @@ PasteDeploy>=1.5.0
 Paste
 sqlalchemy-migrate==0.9.1
 netaddr>=0.7.12
-suds>=0.4
-paramiko>=1.13.0
-posix_ipc
 pyasn1
 Babel>=1.3
 iso8601>=0.1.9
@@ -30,18 +26,13 @@ jsonschema>=2.0.0,<3.0.0
 python-cinderclient>=1.1.0
 python-neutronclient>=2.3.6,<3
 python-glanceclient>=0.14.0
-python-keystoneclient>=0.10.0
 six>=1.7.0
 stevedore>=1.0.0  # Apache-2.0
 websockify>=0.6.0,<0.7
 wsgiref>=0.1.2
 oslo.config>=1.4.0  # Apache-2.0
 oslo.db>=1.0.0,<1.1  # Apache-2.0
-oslo.rootwrap>=1.3.0
-pycadf>=0.6.0
 oslo.messaging>=1.4.0,!=1.5.0,<1.6.0
 oslo.i18n>=1.0.0  # Apache-2.0
-lockfile>=0.8
 simplejson>=2.2.0
 rfc3986>=0.2.0  # Apache-2.0
-oslo.vmware>=0.6.0,<0.9.0               # Apache-2.0
